monkey/tests/unit_tests/infection_monkey/ransomware/test_file_selectors.py

```diff
@@ -53,3 +55,12 @@ def test_directories_not_selected(ransomware_test_data, file_selector):
     selected_files = file_selector(ransomware_test_data)
 
     assert (ransomware_test_data / SUBDIR / HELLO_TXT) not in selected_files
+
+
+def test_ransomware_readme_not_selected(ransomware_target, file_selector):
```

Comment:
Shouldn't we add a test where README.txt get's selected if contents are different?